marlowe-dashboard-client/static/css/main.css

```diff
@@ -7,26 +7,22 @@ body {
   height: 100%;
 }
 
-@layer components {
-  .btn {
-    @apply p-0.5 leading-none focus:outline-none;
-  }
+h2 {
+  @apply font-bold text-lg mb-1;
```

Comment:
I'd suggest against using global margins or paddings in top level components. Positioning is something that should be the responsibility of the parent, not the child

Reply:
I put these in the code at first as arrays of class names. And I found I was adding them to every single element of the type. We can revisit when we've got the actual visual designs to implement. But if an element _always_ has some base styles throughout the site, why not just put that once in the global CSS and forget about it?